packages/crypto/src/crypto.rs

```diff
@@ -10,30 +14,42 @@ use crate::identity_digest::Identity256;
 /// Max length of a message hash for secp256k1 verification in bytes.
 /// This is typically a 32 byte output of e.g. SHA-256 or Keccak256. In theory shorter values
 /// are possible but currently not supported by the implementation. Let us know when you need them.
-pub const MESSAGE_HASH_MAX_LENGTH: usize = 32;
+pub const MESSAGE_HASH_MAX_LEN: usize = 32;
+
+/// Max length of a message for ed25519 verification in bytes.
+/// This is an arbitrary value, for performance / memory contraints. If you need to verify larger
+/// messages, let us know.
+pub const MESSAGE_MAX_LEN: usize = 4096;
```

Comment:
Good approach. Let's be a bit more generous. For database writes we read up to 128 KiB from the contract. We can use this here too.